sources/packages/web/src/assets/css/formio-shared.scss

```diff
@@ -705,3 +705,7 @@
 .formio-component-file ul li.list-group-item {
   border: 1px solid rgba(0, 0, 0, 0.125) !important;
 }
+
+.overflow-visible .v-card {
```

Comment:
any style connected to components of vuetify we add the class to vuetify.scss. Can you please move it there?